### ISSUE TITLE
Getting TypeError: options is undefined on 2nd outgoing call

### ISSUE THREAD

**Issue by BernardoGiordano** [onsip/SIP.js#646]
**Describe the bug**
I'm getting a `TypeError: options is undefined` when I make a call through SIP.js while I have another outgoing call going on. This doesn't happen if I start a call after I ended the one that was ongoing before. Additionally, if I try to make the call after I got the `TypeError`, the call starts correctly.

**Logs**
`TypeError: options is undefined[Ulteriori informazioni] sip-0.13.0.js:24225:9`

**To Reproduce (if possible)**
1. Start a new session through `UA.invite`
2. While the session is active and the call is going on, call `UA.invite` again

**Expected behavior**
Call starts like the first one. Code is the same for every outgoing call that is started.
```
        session = userAgent.invite(number, {
            sessionDescriptionHandlerOptions: {
                constraints: {
                    audio: true,
                    video: false
                }
            }
        });
```

**Environment Information**
 - Firefox 65.0, Ubuntu 18.04
 - sipjs 0.13.0

**Comment by egreenmachine** [comment]
I am unable to reproduce this. Can you include some logs with traceSip? Do you potentially have some custom code in the SDH?

**Comment by BernardoGiordano** [comment]
I will try to reproduce this asap in the upcoming days. Having `traceSip: true` won't give much info about this issue because the call won't even start. I can provide sip logs for the calls immediately after and before the one that causes the crash, but I don't know how helpful will they be.

And nope, I didn't modify SDH's code, I only passed constraints and peerConnectionOptions while setting up the SIP.UA

```
			sessionDescriptionHandlerFactoryOptions: {
				peerConnectionOptions: {
					rtcConfiguration: {
						iceServers: [{urls: /* my stun */}]
					}
				},
				constraints: {
					audio: true,
					video: false
				}
			}
```

**Comment by egreenmachine** [comment]
Full logs would definitely help. You can remove any sensitive information

**Comment by seanbright** [comment]
Probably unrelated, but when looking through the code the other day I came across this:

https://github.com/onsip/SIP.js/blob/master/src/PublishContext.ts#L30-L52

I believe the `options` being referred to here is the constructor argument (`options`) and not the class field (`this.options`).

**Comment by BernardoGiordano** [comment]
Bug is fixed on 0.13.2, thanks
